### DISCUSSION TITLE
第4节课笔记(17班)

### DISCUSSION THREAD

**Discussion by Jianfeng777** [InternLM/Tutorial#277] [笔记]
请大家将第四节课笔记上传到这里哦！

**Comment by l241025097** [comment]
第四节课笔记
![04XTuner大模型单卡低成本微调实战读书笔记](https://github.com/InternLM/tutorial/assets/31737569/11486fcb-ccd9-4613-acbd-ce9dc4258c24)

**Comment by futtttttttttttt** [comment]
https://zhuanlan.zhihu.com/p/677779844

**Comment by zcg-joker** [comment]
[https://blog.csdn.net/m0_55764641/article/details/135589334](https://blog.csdn.net/m0_55764641/article/details/135589334)

**Comment by fine-lab** [comment]
https://blog.csdn.net/SmallZCan_t/article/details/135585513?spm=1001.2014.3001.5501

**Comment by Alisa742** [comment]
https://blog.csdn.net/weixin_53222965/article/details/135590415?spm=1001.2014.3001.5502

**Comment by AxrLH** [comment]
第四课笔记
https://blog.csdn.net/Axr_Samuel/article/details/135586246?csdn_share_tail=%7B%22type%22%3A%22blog%22%2C%22rType%22%3A%22article%22%2C%22rId%22%3A%22135586246%22%2C%22source%22%3A%22Axr_Samuel%22%7D
